src/board.c

```diff
@@ -504,7 +411,7 @@ void Board_LTC6804_GetCellTemperatures(BMS_PACK_STATUS_T * pack_status, uint8_t
 #ifndef TEST_HARDWARE
 #ifdef FSAE_DRIVERS
     if ((msTicks - board_lastThermistorShiftTime_ms) > TIME_PER_THERMISTOR_MS) {
-        board_lastThermistorShiftTime_ms += TIME_PER_THERMISTOR_MS;
+        board_lastThermistorShiftTime_ms = msTicks;
```

Comment:
@jatron I am assuming the old behavior was buggy and am 99% sure this is what you meant to do, please confirm though.